[IAM-5141] add SSO docs for configuring Google IdP (#1758)
saucelabs/sauce-docs: docs/basics/sso/configuring-sso-in-google.md

---
id: configuring-sso-in-google
title: Configuring SSO in Google
sidebar_label: Configuring Google
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<p><span className="sauceDBlue">Enterprise Plans only</span></p>

> Sauce Labs SAML app is not available in [Google SAML catalog](https://support.google.com/a/topic/7417510) as Google has paused processing new requests from App Developers. The only option for you to integrate with Sauce Labs is to set up a [custom SAML application](https://support.google.com/a/answer/6087519).

Reviewer: ```suggestion
Sauce Labs SAML app is not available in [Google SAML catalog](https://support.google.com/a/topic/7417510) as Google has paused processing new requests from App Developers. You can integrate with Sauce Labs by setting up a [custom SAML application](https://support.google.com/a/answer/6087519).
```
